src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -11,7 +12,7 @@
 
 public class RepositoryApiClientImpl implements RepositoryApiClient, AutoCloseable {
     private Map<String, String> defaultHeaders;
-    private UnirestInstance httpClient;
+    private static UnirestInstance httpClient;
```

Comment:
since this is static, does that mean if I create two `RepositoryApiClientImpl`, the httpClient is shared? And if I call `setDefaultRequestHeaders` on one `RepositoryApiClientImpl`, it will also affect the second `RepositoryApiClientImpl`?

Reply:
Hi @jason-chan-lf , it has been changed to non-static.

Reply:
@sxu1998 I think we discussed this before, there's no need to make it static. Let's change it back.

Reply:
ok done